examples/java8/src/main/java/org/apache/beam/examples/complete/game/UserScore.java

```diff
@@ -216,6 +218,9 @@ public void processElement(ProcessContext c) {
 
   /**
    * Run a batch pipeline.
+   *
+   * @param args The main arguments.
```

Comment:
Also for `@param` I don't think it is that helpful to require the annotations. Usually it is better to have sentences that actually describe what is going on. Code review is enough to make sure that when `@param` is needed that someone will add it, but that is maybe 10% of the time or less IMO.